tests/Integration/Behaviour/Features/Scenario/Product/update_prices.feature

```diff
@@ -0,0 +1,141 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-prices
+@reset-database-before-feature
+@update-prices
+Feature: Update product price fields from Back Office (BO).
+  As a BO user I want to be able to update product fields associated with price.
+
+  Scenario: I update product prices
+    Given I add product "product1" with following information:
+      | name       | en-US:magic staff   |
+      | is_virtual | false               |
+    And product "product1" should have following values:
+      | price              | 0           |
+      | ecotax             | 0           |
+      | tax rules group    |             |
```

Comment:
Not `0` ?

Reply:
its not id anymore, so its empty/none/not assigned. ( can write smth like that if its better?)
`if ($data['tax rules group'] === 'none') { assert as 0 }`